main.go

```diff
@@ -56,11 +56,10 @@ type apiStatus struct {
 
 // Init setup the apiKey
 func Init(config Config) *TMDb {
-	internalConfig := new(tmdbConfig)
```

Comment:
Did you mean to remove this line? It looks like only the other change is actually needed.

Reply:
But this internalConfig useless，Even if you don't delete it, it won't have any impact.Of course, the best way is for each client to initialize and maintain the proxy information by itself, but it is OK if the library provides a default.